### PR TITLE
Issue #17 - Add some lateral margin to Slider on non-small devices

### PR DESCRIPTION
**Pull request by mblaul**
- Adding a small amount of left and right margin to the slider on large displays
- Removing style props from Slider component so class styles are not overridden

I only added a small amount of margin, if you want something more extreme let me know and I can update it!
spacing

### CODE REVIEW

File: app/src/Widget/List.jsx [hunk 1]
```diff
@@ -72,6 +72,17 @@ const useStyles = (theme) => ({
...
   },
+  // Add some lateral space for the slider on larger displays
+  sliderRoot: {
+    [theme.breakpoints.up('md')]: {
+      marginLeft: theme.spacing(4),
+      marginRight: theme.spacing(4),
+    },
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(1),
+      marginRight: theme.spacing(1),
```
Comment (emibcn) [inline app/src/Widget/List.jsx:83]: I've tested with just `1` and it overlaps with lateral buttons. Looks like the minimal for small devices should be `3`.

File: app/src/Widget/List.jsx [hunk 2]
```diff
@@ -72,6 +72,17 @@ const useStyles = (theme) => ({
       },
     }
   },
+  // Add some lateral space for the slider on larger displays
+  sliderRoot: {
+    [theme.breakpoints.up('md')]: {
+      marginLeft: theme.spacing(4),
+      marginRight: theme.spacing(4),
+    },
+    [theme.breakpoints.down('sm')]: {
```
Comment (emibcn) [inline app/src/Widget/List.jsx:81]: To have all devices controlled, you should be using `down('md')` here

**Review by emibcn — CHANGES_REQUESTED** [review]
Looks almost great, just a couple of comments in the review.

**Comment by mblaul** [comment]
@emibcn Thanks for the review and comments. Sorry about that!
I have updated this PR to address those points.

**Comment by emibcn** [comment]
[Thanked!](https://github.com/emibcn/covid#thanks-to)

**Pull request merged**